docs/content/Configuration/compatibility.md

```diff
@@ -12,6 +12,12 @@ version dependencies between the two projects. Below are the operator releases a
 
 | Operator Release   |      Container Release      | Postgres | PgBackrest Version
 |:----------|:-------------|:------------|:--------------
+| 4.4.1 | 4.4.1  | 12.4 | 2.27 |
```

Comment:
On the main branch, should this table contain every released version (e.g. 4.2.2, the upcoming 4.3.3)?

Maybe instead we link to another branch of the docs?

Reply:
Maybe, but we start running into recursive problems.